libraries/abstractions/common_io/test/test_iot_i2s.c

```diff
@@ -572,7 +575,11 @@ TEST( TEST_IOT_I2S, AFQP_IotI2SIoctl )
        xTestConfig.xI2SWsPolarity = eI2SFallingEdge;
 
        /* Validate different frequencies */
-       xTestConfig.ulI2SFrequency++;
+       if( xTestConfig.ulI2SFrequency != ulFreqVals[0] ) {
+          xTestConfig.ulI2SFrequency = ulFreqVals[0];
+       } else {
+          xTestConfig.ulI2SFrequency = ulFreqVals[1];
+       }
```

Comment:
I don't understand what this "if block" really wants to test.

Reply:
If the running I2S Frequency is the value at index [0] of ulFreqVals, change it to the other one at index [1] before you apply it with SetConfig.
But if it's different, use the one at index [0].

We want to set a different parameter than the running configuration to see it is reflected back in the ioctl GetConfig call after we do SetConfig.

Reply:
At line 578-582, you set the xTestConfig, but there is no eI2SSetConfig ioctl call after it.
At line 583, xTestConfig is passed into ioctl eI2SGetConfig and its value will be overwritten. Then line 578-582 is meaningless.

Reply:
Good catch, many of the `eI2SGetConfig` used should be `eI2SSetConfig`. I'll update it.

Reply:
@qiutongs , the commit is updated now. Please have a look.